3-networks-hub-and-spoke/modules/base_env/main.tf

```
@@ -207,7 +208,7 @@ module "restricted_shared_vpc" {
   environment_code                  = var.environment_code
   access_context_manager_policy_id  = var.access_context_manager_policy_id
   restricted_services               = local.restricted_services
-  members                           = distinct(concat(["serviceAccount:${local.networks_service_account}", "serviceAccount:${local.projects_service_account}"], var.perimeter_additional_members))
+  members                           = distinct(concat(["serviceAccount:${local.networks_service_account}", "serviceAccount:${local.projects_service_account}", "serviceAccount:${local.environments_service_account}"], var.perimeter_additional_members))
```

Could you please brake the line into multiple lines? it has become a bit long now.

```suggestion
  members = distinct(concat([
    "serviceAccount:${local.networks_service_account}",
    "serviceAccount:${local.projects_service_account}",
    "serviceAccount:${local.environments_service_account}",
  ], var.perimeter_additional_members))
```

this change will also trigger formatting changes in the following lines:

```hcl
  private_service_cidr       = var.restricted_private_service_cidr
  private_service_connect_ip = var.restricted_private_service_connect_ip
  ingress_policies           = var.ingress_policies
  egress_policies            = var.egress_policies
  bgp_asn_subnet             = local.bgp_asn_number
  default_region1            = var.default_region1
  default_region2            = var.default_region2
  domain                     = var.domain
  mode                       = "spoke"
```